composer.json

```diff
@@ -26,8 +26,8 @@
   },
   "require-dev": {
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-    "giacocorsiglia/wordpress-stubs": "^5.1",
-    "infinum/coding-standards-wp": "0.4.3"
+    "infinum/coding-standards-wp": "0.4.3",
+    "php-stubs/wordpress-stubs": "^5.5"
```

Comment:
Do you have tests? What do you need stubs for?

Reply:
For when developing the plugin, I open the root of the plugin (because it's installed in the actual project) and in that case my editor doesn't know about any WordPress function and phpcs goes crazy. These stubs fix that issue.

Reply:
Hmmm, haven't thought about that tbh. That is actually really smart! I always open my projects in the root of WP so that PHPStorm can autocomplete everything.

Do you have documentation about WP specific functions with stubs? Like when you hover over them?

Reply:
> Do you have documentation about WP specific functions with stubs? Like when you hover over them?

Yup, everything is there, description, parameter type hints / description, etc (all except the actual body of function haha)

> I always open my projects in the root of WP so that PHPStorm can autocomplete everything.

Usually me too but here in the root I have a different repository (project I'm working on) and my Gitlens doesn't really know how to handle those 2 repos meaning I have to do it like this (have 2 editor window for project, another for the plugin).